string status
string current_item
int32 total_number